src/reference/modules/administration/pages/security.adoc

```diff
@@ -1,178 +1,126 @@
-= Network security
+= Rudder security & hardening
 
-Communication between servers and nodes uses two channels:
+Rudder provides secure defaults whenever possible, but depending on your use case you can
+add additional configurations to make your Rudder installation more secure.
 
-* HTTPS from nodes to servers
-* Custom protocol wrapped in TLS on tcp/5309, from nodes to server to
-download policies and (optionally) from server to nodes to trigger runs.
+== Server
 
-Both enforce using TLS 1.2+.
-
-We recommend using a VPN for all communications between node and server
-if on a public network. If it's not the case, it's necessary to follow the following
-guide to improve security level.
-
-== Authentication mechanisms
-
-=== Credentials
-
-On all agents and relays, and servers:
-
-* `/var/rudder/cfengine-community/ppkeys/localhost.priv`, the node's private key, that allows the node to download its policies. It is a 4096-bit RSA key.
-* `/var/rudder/cfengine-community/ppkeys/localhost.pub`, the node's public key
-* `/opt/rudder/etc/agent.cert`, the node's certificate. It is embedded in inventories and signs inventories and compliance reports.
-
-Only on servers and relays:
-
-* `/opt/rudder/etc/rudder.key`, the HTTP server's private key. It is a 2048-bit RSA key by default.
-* `/opt/rudder/etc/rudder.crt`, the HTTP server's certificate chain. It is generated as a self-signed certificate at server installation. See procedure below to replace it by a valid one.
-
-=== Policy update and remote run (tcp/443)
-
-This protocol is used for:
-
-* Policy download on Linux and AIX nodes
-* Remote run from policy servers to nodes
+=== HTTPS
 
-It uses the key pairs in `/var/rudder/cfengine-community/ppkeys/` to authenticate the node and server during
-communication.
+* By default, all HTTPS traffic is handled by the same virtual host in Apache httpd. To allow hardening the connection security, a first step is to split public Web/API from internal node-server communication. You can do so by commenting the default virtual host in `/etc/apache2/sites-enabled/rudder.conf` or `/etc/httpd/conf.d/rudder.conf` (depending on the distribution), and uncommenting the two separate configurations. You only need a way to make a difference in the configuration, in general with a different port (which allows applying different firewall rules too) or a specific domain for Web/API.
```

Comment:
You only need a way to make a difference in the configuration, in general with a different port (which allows applying different firewall rules too) or a specific domain for Web/API.
->
You need to define a method to distinguish both configurations, in general with a different port (which allows applying different firewall rules too) or a specific domain for Web/API.